plugins/rails/rails.plugin.zsh

```diff
@@ -68,7 +68,9 @@ alias rmd='rails middleware'
 alias rn='rails notes'
 alias rp='rails plugin'
 alias rr='rails routes'
-alias rrg='rails routes | grep'
+alias rre='rails routes --expanded'
+alias rrg='rails routes -g'
+alias rrc='rails routes -c'
```

Comment:
@joshRpowell Is it possible to use the "extended version" of the flags `-g` and `-c`? Like you did in line 71 with `--expanded`. This would make the documentation and use more intuitive, in case a user is not so familiar with the flags of the `rails routes` command.